torch/_inductor/codegen/cpp.py

```diff
@@ -1192,7 +1192,6 @@ def __init__(self, args, num_threads):
         self.poststores = IndentedBuffer()
         self.num_threads = num_threads  # num_threads the kernel specialized for
         self.reduction_omp_dec: Dict[Tuple[str, str], str] = {}
-        self._load_mask = None
```

Comment:
The load mask is relevant to C++ kernels. When you have vectorized loads, it can mask out individual elements in a vector.

Reply:
I've moved this variable to `common.py::Kernel` as both `CppKernel` and `TritonKernel` had a version of it

Reply:
Okay but the point still stands that `load_mask` shouldn't be a no-op on CPU.

Reply:
Oh, for CPU, the `_load_mask` is initialized in the `ops.masked` call here: https://github.com/pytorch/pytorch/blob/6e3a7473cf45229a7a7b2e98611991f69d1c1437/torch/_inductor/codegen/cpp.py#L1204
It is not a no-op on CPU.